samcli/commands/local/lib/generated_sample_events/events/sagemaker/AnnotationConsolidation.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": "2018-10-16",
+    "labelingJobArn": "arn:{{{partition}}}:sagemaker:{{{region}}}:{{{account_id}}}:labeling-job/{{{labeling_job_name}}}",
+    "labelAttributeName": "{{{label_attribute_name}}}",
+    "roleArn" : "aws:{{{partition}}}:iam::{{account_id}}:role/{{{execution_role}}}",
+    "payload": {
+        "s3Uri": "s3://{{{bucket}}}/{{{labeling_job_name}}}/annotations/worker_response/{{{iteration_object_timestamp}}}.json"
```

Comment:
Similar here: It could make it easier for customers to just provide the `bucket/labeling_job_name` as one parameter instead of 2. Thoughts?

Reply:
Good idea. I'll go back through and make the params match what's in CreateLabelingJob, since in some cases, the creator isn't specifying a s3 bucket, but rather a full s3 url.